src/ontology/mondo-ingest.Makefile

```diff
@@ -355,23 +358,43 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# Builds tmp/mondo/ and rebuilds mondo.owl and mondo.sssom.tsv, and stores hash of latest commit of mondo repo main branch in tmp/mondo_repo_built
 tmp/mondo_repo_built:
 	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
 		rm -rf ./mondo/ &&\
 		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
 		cd mondo/src/ontology &&\
 		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
-		cd ../../../../ &&\
-		touch $@; fi
-
-$(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
-	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@
-
-$(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
-	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
-
-reports/mirror_signature-mondo.tsv: tmp/mondo.owl
-	$(ROBOT) query -i $< --query ../sparql/classes.sparql $@
+		latest_hash=$$(git rev-parse origin/master) &&\
+		echo "$$latest_hash" > $@ &&\
+		cp $@ mappings/mondo.sssom.tsv mondo.owl ../../../; fi
+
+# Triggers a refresh of tmp/mondo/ and a rebuild of mondo.owl and mondo.sssom.tsv, only if mondo repo main branch has new commits
```

Comment:
## Doc comments
Added comments to `refresh-mondo-clone` and `tmp/mondo_repo_built`, explaining what they do.